### PR TITLE
DEV-15092 Add configurable timeout for HDFS access

### PR DESCRIPTION
**Pull request by nkapinos**
@jleealpine @kevinmtrowbridge @midnighteuler Code review?  This is causing notebooks to fail when reading large files..especially if Chorus is on AWS and HDFS is at Alpine.

### CODE REVIEW

**Comment by nkapinos** [comment]
@kevinmtrowbridge  Code review?

**Comment by kevinmtrowbridge** [comment]
Hi Nate -- apologies for the delay. I will take a look on Monday.

File: components/core/app/services/hdfs/query_service.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ module Hdfs
   PermissionDeniedError = Class.new(StandardError)
 
   JavaHdfs = com.emc.greenplum.hadoop.Hdfs
-  JavaHdfs.timeout = 5
+  JavaHdfs.timeout = ChorusConfig.instance['hdfs_read_timeout_seconds']
```
Comment (kevinmtrowbridge) [inline components/core/app/services/hdfs/query_service.rb]: Hi @nkapinos -- thanks for being patient.

This looks pretty straightforward.

One thing, here -- use this notation, to actually use the getting method that you create in ChorusConfig: `ChorusConfig.instance.hdfs_read_timeout_seconds` -- using the square brackets notation, will just read the value out of the instance, skipping the getter method.

Comment (kevinmtrowbridge) [inline components/core/app/services/hdfs/query_service.rb]: I will go ahead and make the change so as to save a little time.

**Comment by kevinmtrowbridge** [comment]
This looks good to me, @nkapinos!  I will let you merge it in.

**Pull request merged**